lxd/main.go

```diff
@@ -633,6 +635,10 @@ func setupLXD() error {
 	}
 
 	if *argAuto {
+		if *argStorageBackend == "" {
```

Comment:
"lxd init" is meant to "initialize", so it resets all the keys and sets them all again.
It is not meant to be run once you have any image imported or container created, in fact, it won't let you do so.